### PR TITLE
FIx install of yml file for audioseam filter

### PR DESCRIPTION
**Pull request by j-b-m**
The audioseam yml file currently doesn't install because the CMakeLists.txt is broken

### CODE REVIEW

File: src/modules/core/CMakeLists.txt [hunk 1]
```diff
@@ -75,46 +75,6 @@ set_target_properties(mltcore PROPERTIES LIBRARY_OUTPUT_DIRECTORY "${MLT_MODULE_
...
-  producer_melt.yml
-  producer_noise.yml
-  producer_timewarp.yml
-  producer_tone.yml
-  transition_composite.yml
-  transition_luma.yml
-  transition_matte.yml
-  transition_mix.yml
-  loader.dict
-  loader.ini
```
Comment (bmatherly) [inline src/modules/core/CMakeLists.txt:118]: Should loader.dict and loader.ini stay in the list? Or do they get installed some other way?

Comment (ddennedy) [reply src/modules/core/CMakeLists.txt:118]: Thanks Brian those do need to be installed, and this commit broke that. And these are much more critical data files than YAML! Can you fix it please @j-b-m ? I’m on vacation.

**Comment by ddennedy** [comment]
Those are target dependencies and make the files appear in IDEs such as Qt Creator. There is a separate install command where these are not included. I also checked last daily (macos) build log here in Actions, and see there is no longer lines showing these two files get installed. Fix it!!

**Comment by j-b-m** [comment]
Fixed, sorry for this.

**Pull request merged**